contrib/crio-wipe/README.md

```diff
@@ -0,0 +1,27 @@
+# crio-wipe
+
+crio-wipe is a program that reads CRI-O's version file, and compares it to the output of crio --version.
+If the version is deemed old enough, crio-wipe wipes container storage on the node.
+If there is no version file present, or the version file is malformatted, crio-wipe also will wipe the storage.
+
+
+crio-wipe returns 0 on success and 1 on error. If a wipe happened, it will be noted in stdout. Otherwise, crio-wipe will not wipe silently
```

Comment:
I'd drop "Otherwise, crio-wipe will not wipe silently.  It's redundant,  in the previous statement you just noted that it will be noted.